api/proto/teleport/legacy/types/types.proto

```diff
@@ -6170,6 +6170,8 @@ message PluginStatusV1 {
     // gitlab is the status details for the Gitlab plugin.
     PluginGitlabStatusV1 gitlab = 4;
     PluginEntraIDStatusV1 entra_id = 5;
+    // Okta holds status details for the Okta plugin
+    PluginOktaStatusDetailsV1 okta = 7;
```

Comment:
suggestion: It is already defined under `oneof details {` so we can omit "Details"? Gitlab and EntraID does the same above.
```suggestion
    PluginOktaStatusV1 okta = 7;
```